package.json

```
@@ -8,11 +8,8 @@
     ]
   },
   "scripts": {
-    "apidoc": "yarn clean && yarn build && yarn apidoc:extract && yarn apidoc:generate",
-    "apidoc:ci": "yarn apidoc:extract && yarn apidoc:generate",
-    "apidoc:extract": "yarn workspaces run apidoc:extract && node scripts/fix-extracted-api-files.js",
-    "apidoc:generate": "api-documenter markdown -i docs -o docs/dist && yarn cp scripts/gh-pages_config.yml docs/dist/_config.yml && node scripts/enhance-doc-index-md.js",
-    "apidoc:gh-pages": "gh-pages -d docs/dist -m 'Updates [skip CI]'",
+    "typedoc": "yarn clean && yarn build && typedoc --skipErrorChecking",
+    "typedoc:gh-pages": "yarn typedoc && gh-pages -d ./docs -m 'Updates [skip CI]'",
```

Just change the commit message:

```suggestion
    "typedoc:gh-pages": "yarn typedoc && gh-pages -d ./docs -m 'docs: updates documentation [skip CI]'",
```